src/backend/joanie/core/templates/issuers/contract_definition.html

```diff
@@ -137,9 +137,12 @@ <h1>{{ contract.title }}</h1>
                       </div>
                   </div>
               </section>
-              {% if syllabus or student.payment_schedule %}
+              {% if contract.appendix or syllabus or student.payment_schedule %}
```

Comment:
What if there is a **Syllabus** and the **appendix** , is it problematic if the syllabus does not show up ?